src/etl/gwas_ingest/study_ingestion.py

```diff
@@ -294,30 +486,84 @@ def ingest_gwas_catalog_studies(
     # Read GWAS Catalogue raw data
     gwas_studies = read_study_table(etl, study_file)
     gwas_ancestries = parse_ancestries(etl, ancestry_file)
+    ss_studies = get_sumstats_location(etl, summary_stats_list)
 
+    # Sample size extraction is a separate process:
     study_size_df = extract_discovery_sample_sizes(gwas_studies)
-    ss_studies = get_sumstats_location(etl, summary_stats_list)
 
-    studies = (
+    # Mapping ancestries of the discover sampe description to GnomAD:
+    # gnomad_ancestries = parse_gnomad_ancestries(
```

Comment:
Is this supposed to be commented?

Reply:
Yes, the gnomad mapping is no longer part of the study ingestion. I'm going to remove the commented lines.